index.html

```diff
@@ -259,5 +259,18 @@ <h6>Skills</h6>
         </li>
       </ul>
     </section>
+    <footer class="contact-section">
+      <div class="contact-box">
+        <h3 class="h3-contact">Contact me</h3>
+        <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <form class="contact-form" action="https://formspree.io/f/mdobgbjy" method="POST">       
+        <input type="text" placeholder="Onomeh"  name="username" required/>
```

Comment:
- [ ] As stated in the project requirements: Please ensure that the name input accepts up to 30 chars. You can do this by adding the `maxlength="30"` attribute.